src/main/java/com/OrganizationController.java

```diff
@@ -69,39 +47,32 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
             }else{
                 response.setStatus(HttpServletResponse.SC_NOT_FOUND);
             }
-
         }
     }
 
-/*
+    @Override
     protected void doDelete(HttpServletRequest request, HttpServletResponse response) throws ServletException,
             IOException {
 
         System.out.println("DELETE");
```

Comment:
Remove all souts

Reply:
Thank you @jayasanka-sack . A small thing I need to clarify.
Souts means these @Override symbols right?

Reply:
nope. System.out.printlns :)

Reply:
Fixed with 2b5516c

Reply:
Done. Fix with https://github.com/pasindur99/Organizations-backend/pull/5/commits/072d8ed5489e0bd8dc51ed1a72db35cd2f08af6b